boards/riscv/it82xx2_evb/board.cmake

```diff
@@ -0,0 +1,3 @@
+set(SUPPORTED_EMU_PLATFORMS renode)
```

Comment:
Which resc/repl files is this board using? Because this file is missing the `RENODE_SCRIPT` variable.

Reply:
West flash and west debug are not currently supported, so we can remove this file.